.drone/drone.jsonnet

```diff
@@ -339,7 +339,7 @@ local manifest(apps) = pipeline('manifest') {
       run('LogQL', ['go test -mod=vendor -bench=Benchmark -benchtime 20x -timeout 120m ./pkg/logql/'])
     ],
     trigger: {event: ['cron'], cron: ['loki-bench']},
-    //when: condition('include').tagMain,
```

Comment:
hmm, I thought that crons always used the main branch 🤔

Reply:
Ok. Then I'll remove this.